src/ExpenseTrackerGrupo2/Persistence/Database/Scripts/init.sql

```diff
@@ -0,0 +1,77 @@
+CREATE TABLE IF NOT EXISTS Users (
+    Id SERIAL PRIMARY KEY,
+    Name VARCHAR(100) NOT NULL,
+    Email VARCHAR(100) NOT NULL UNIQUE,
+    PasswordHash VARCHAR(255) NOT NULL,
+    CreatedAt TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+);
+
+CREATE TABLE IF NOT EXISTS Expenses (
+    Id SERIAL PRIMARY KEY,
+    UserId INT NOT NULL,
+    Amount DECIMAL(10, 2) NOT NULL,
+    Description VARCHAR(255),
+    Source VARCHAR(100),
+    Month INT NOT NULL,
+    GoalAmount DECIMAL(10, 2),
+    Category VARCHAR(100),
+    Date DATE NOT NULL,
+    CreatedAt TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    FOREIGN KEY (UserId) REFERENCES Users(Id)
+);
+
+CREATE TABLE IF NOT EXISTS Incomes (
+    Id SERIAL PRIMARY KEY,
+    UserId INT NOT NULL,
+    Amount DECIMAL(10, 2) NOT NULL,
+    Date DATE NOT NULL,
+    CreatedAt TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    FOREIGN KEY (UserId) REFERENCES Users(Id)
+);
+
+CREATE TABLE IF NOT EXISTS Budgets (
+    Id SERIAL PRIMARY KEY,
+    UserId INT NOT NULL,
+    BudgetAmount DECIMAL(10, 2) NOT NULL,
+    CreatedAt TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    FOREIGN KEY (UserId) REFERENCES Users(Id)
+);
+
+CREATE TABLE IF NOT EXISTS Goals (
+    Id SERIAL PRIMARY KEY,
+    UserId INT NOT NULL,
+    Deadline DATE NOT NULL,
+    CurrentAmount DECIMAL(10, 2) NOT NULL,
+    CreatedAt TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    FOREIGN KEY (UserId) REFERENCES Users(Id)
+);
+
+INSERT INTO Users (Name, Email, PasswordHash, CreatedAt) 
```

Comment:
Don't forget to update inserts according the types given by the backend.

Reply:
The tables are  according to document Expense_tracker_api_requirements_document in database design part, pls review that part.

Reply:
Would you mind to review the document again please?

Reply:
Expenses don't have Month, GoalAmount, Source columns